src/features.ts

```diff
@@ -131,11 +144,9 @@ const Features = {
   },
 
   featureFailed (featureName, err) {
-    log(`[RemoteStorage] [FEATURE ${featureName}] initialization failed (${err})`);
-    this.featureDone();
+    throw new Error(`Feature initialization failed for "${featureName}": ${err}`);
```

Comment:
I'm not sure about this change. Wouldn't the whole library initialization fail this way, instead of one particular feature that would be missing? E.g. when IndexedDB initialization failed, the whole library wouldn't work instead of just falling back to localStorage.

Reply:
Good catch! I think you're right.